lisa/tools/mv.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Mv(Tool):
+    @property
+    def command(self) -> str:
+        return "mv"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def move_file(self, src_path: str, dest_path: str, sudo: bool = False) -> None:
+        self.run(f"{src_path} {dest_path}", sudo=sudo, force_run=True, shell=True)
```

Comment:
If a file exists and uses force_run, the command will execute with error, it won't impact the run, since here we don't set expected error code. but the existing file won't be overwritten
We need to think about the usage of this method, we can provide an overwrite option, and use mv with -f option.

Reply:
I've gone ahead and added the overwrite functionality as described here and I've removed the force_run arg as well. Is this an acceptable change to address your concerns?

Reply:
force_run must be here to make sure it actually executed instead of just taking the cached output.

Reply:
I've returned the force_run variable, I misunderstood its use.